drivers/cas_ce_data/app/models/cas_ce_data/synthetic/event.rb

```diff
@@ -43,7 +43,12 @@ def self.remove_orphans
     end
 
     def self.add_new
-      new_events = CasCeData::GrdaWarehouse::CasReferralEvent.where.not(id: self.select(:source_id))
+      # Replaced the following with a lateral join for performance, leaving temporarily for easy reversal
+      # new_events = CasCeData::GrdaWarehouse::CasReferralEvent.where.not(id: self.select(:source_id))
+      existing = where('"cas_referral_events"."id" = source_id').select(:source_id).limit(1)
+      new_events = CasCeData::GrdaWarehouse::CasReferralEvent.
+        joins("LEFT JOIN LATERAL (#{existing.to_sql}) findings ON true").
+        where('findings.source_id is NULL')
```

Comment:
I think this file snuck in from somewhere else.  @dtgreiner was this intentional?

Reply:
@eanders including this code was not intentional. I'm guessing it snuck in when i ran into an issue during a branch swap.